profiles/govcms_saas.yml

```diff
@@ -88,3 +88,23 @@ checks :
     max_size : 5
     warning_size : 2
   \SiteAudit\Check\Phantomas\InPage404s : {}
+
+  # Field specific checks.
+  \SiteAudit\Check\D7\EntityReferenceAutocomplete:
+    threshold: 100
+
+  # Module configuration checks.
+  \SiteAudit\Check\D7\ModuleSettings:
+    label: 'Login Security'
+    machine_name: 'login_security'
+    enabled: true
+    settings:
+      \SiteAudit\Settings\VariableIterator:
```

Comment:
Is it useful to have pluggable iterators? I think this would make the most sense if this just was the iterator that all moduleSettings checks use. Thoughts?